sass/layout/_no-sidebar.scss

```diff
@@ -1,5 +1,15 @@
 .no-sidebar {
 
+	.site {
+		display: grid;
+		grid-template-columns: auto;
+		grid-template-areas:
+			"header"
+			"main"
+			"sidebar"
```

Comment:
The `.no-sidebar` class is only added when there is no sidebar, so the "sidebar" template area has no effect here and thus it must be removed.

Reply:
Right! sidebar needs to remove from grid-template-area.